src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -29,6 +29,7 @@
     "suggest": {
         "symfony/yaml": "",
         "symfony/config": "",
+        "symfony/expression-language": "",
```

Comment:
the explanation about why the package is suggested should not be empty. Otherwise the suggestion is quite useless